internal/licenses/context.go

```diff
@@ -0,0 +1,18 @@
+package licenses
+
+import (
+	"context"
+)
+
+type licenseScannerKey struct{}
```

Comment:
This is unexported and internal so that callers can't mess with the scanner value in the context? And the getter and setter are exported but internal so that only code that's part of Syft can get or set the context?

Just trying to make sure I understand.

Reply:
all correct 👍